Assets/HoloToolkit/Input/Scripts/InputSources/EditorInputSource.cs

```diff
@@ -49,7 +49,7 @@ public ButtonStates()
 
         public bool SupportsPosition;
         public bool SupportsRotation;
-        public bool SupportsPointing;
```

Comment:
That's true! I renamed it to better show that there's more than one kind of ray, instead of just a pointing ray. Though, I don't know how often the grip ray will be used or how useful it is. I'm happy to revert!